Source/Core/LMCHFSSResponseFileHandler.hh

```diff
@@ -26,6 +26,7 @@ namespace locust
         // Member functions
         virtual bool Configure( const scarab::param_node& aNode);
         virtual bool ReadHFSSFile();
+	virtual bool GenerateAnalyticTFtoFIR(double initialFreq, std::vector<std::complex<double>> tfArray);
```

Comment:
You are correct regarding the double definition, so I've removed the original instance of it. I tried to match the structure of the original method that reads in the TF file directly (defined through `ReadHFSSFile()` in this class), which is why it's defined here, although it could be defined in TFFileHandlerCore directly.